data-processor/src/main/java/io/micronaut/data/processor/mappers/jpa/jx/TableAnnotationMapper.java

```diff
@@ -52,7 +52,7 @@ public List<AnnotationValue<?>> map(AnnotationValue<Annotation> annotation, Visi
         final AnnotationValueBuilder<MappedEntity> builder = AnnotationValue.builder(MappedEntity.class);
         annotation.stringValue("name").ifPresent(builder::value);
         annotation.stringValue(SqlMembers.CATALOG).ifPresent(catalog -> builder.member(SqlMembers.CATALOG, catalog));
-        annotation.stringValue(SqlMembers.SCHEMA).ifPresent(catalog -> builder.member(SqlMembers.SCHEMA, catalog));
+        annotation.stringValue("schema").ifPresent(schema -> builder.member("schema", schema));
```

Comment:
Why not use `SqlMembers.SCHEMA`?

Reply:
Due to consistensy reasons. SqlMembers are meta fields, which "schema" is no longer. Also SqlMembers#schema could be removed in the future.